app/main/forms.py

```diff
@@ -1461,30 +1461,32 @@ def validate_address(self, field):
             raise ValidationError(f"Address must be no more than {PostalAddress.MAX_LINES} lines long")
 
         if address.has_invalid_country_for_bfpo_address:
-            raise ValidationError("The last line of a BFPO address must not be a country.")
+            raise ValidationError(
+                "The last line of a British Forces Post Office (BFPO) address cannot be the name of a country"
+            )
 
         if not address.has_valid_last_line:
             if self.allow_international_letters:
-                raise ValidationError("Last line of the address must be a UK postcode or another country")
+                raise ValidationError("The last line of the address must be a UK postcode or the name of a country")
             if address.international:
                 raise ValidationError("You do not have permission to send letters to other countries")
             raise ValidationError("Last line of the address must be a real UK postcode")
 
         if address.has_invalid_characters:
             raise ValidationError(
-                "Address lines must not start with any of the following characters: @ ( ) = [ ] ” \\ / , < > ~"
+                'Address lines cannot start with any of the following characters: @ ( ) = [ ] " " \\ / , < > ~'
```

Comment:
Should the quote marks be ‘smart’ quotes?

> “ ”

Reply:
These are the exact characters the validation is checking:
https://github.com/alphagov/notifications-utils/blob/3e6c6e018163c189c8aaf753cd423aab35903809/notifications_utils/postal_address.py#L30

So it’s a straight double quote, not a smart quote. Either way we shouldn’t have the quote repeated:
```suggestion
                'Address lines cannot start with any of the following characters: @ ( ) = [ ] " \\ / , < > ~'
```

Ideally this error message would reference the definition in code so it updates automatically but that doesn’t have to be for this PR.

Reply:
Made a PR to work on this here: https://github.com/alphagov/notifications-utils/pull/1071

Reply:
Fixed in [257434f](https://github.com/alphagov/notifications-admin/pull/4879/commits/257434fd3253e21cab4fd209d6a4108aa1e1a30f)